ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/documentation/devices/DC1-BL1A.md

```diff
@@ -690,9 +690,9 @@ ASN Notation: asplain
 
 ##### EVPN Peer Groups
 
-| Peer Group | Activate | Encapsulation |
-| ---------- | -------- | ------------- |
-| EVPN-OVERLAY-PEERS | True | default |
+| Peer Group | Activate | Route-map In | Route-map Out | Encapsulation |
+| ---------- | -------- | ------------ | ------------- | ------------- |
+| EVPN-OVERLAY-PEERS | True |  - | - | default |
 
```

Comment:
getting these unexpected diffs in multiple files, this needs to be fixed in `devel`.
is it okay to keep it in this PR or shall I do a separate PR for this?

Reply:
let's keep them here that was missed in a previous PR becayse of CI not running these. Will adjust CI later to not miss it again.